### ISSUE TITLE
Randomly stops working

### ISSUE THREAD

**Issue by ThinkSalat** [WootingKb/wooting-macros#134]
It randomly stops working - not recognizing or inputting macros at all. Not sure what else to provide, it has happened for the simplest macros. It simply doesn't catch when you're entering the input it's supposed to be listening for

**Comment by ThinkSalat** [comment]
Restarting the gui doesn't work, you must go to task manager and end all wootamation processes. Then restarting seems to work

**Comment by medzernik** [comment]
Is this on 1.0.2?

**Comment by ThinkSalat** [comment]
Yes it is. I made sure to check I was on the latest version. I also use AHK
but there's no interference going on there in terms of keybindings.

**Comment by medzernik** [comment]
Might be related to #103 or #109, but those should have been fixed
